src/ViewInterface.coffee

```diff
@@ -0,0 +1,6 @@
+define 'ViewInterface',
+[],
+() ->
+    # ViewInterface is an interface for component classes
+    class ViewInterface
+        init: -> {}
```

Comment:
I still don't understand why do we need this class. Will make a call and you explain me

Reply:
ok